src/qt/pivx/topbar.cpp

```diff
@@ -486,13 +486,13 @@ void TopBar::updateBalances(const CAmount& balance, const CAmount& unconfirmedBa
                             const CAmount& watchOnlyBalance, const CAmount& watchUnconfBalance, const CAmount& watchImmatureBalance){
 
     CAmount nLockedBalance = 0;
-    if (!walletModel) {
+    if (walletModel) {
         nLockedBalance = walletModel->getLockedBalance();
     }
 
     // PIV Balance
-    //CAmount nTotalBalance = balance + unconfirmedBalance;
-    CAmount pivAvailableBalance = balance - immatureBalance - nLockedBalance;
+    CAmount nTotalBalance = balance + unconfirmedBalance + immatureBalance;
+    CAmount pivAvailableBalance = nTotalBalance - unconfirmedBalance - immatureBalance - nLockedBalance;
```

Comment:
why not simply `CAmount pivAvailableBalance = balance - nLockedBalance;` ?

Reply:
updated